documentation/api/parameters.md

```diff
@@ -0,0 +1,167 @@
+
+### Was this documentation helpful? [Share feedback](https://www.research.net/r/DGDQWXH?src=documentation%2Fapi%2Fparameters)
+
+# Parameters - Post (experimental feature)
+
+Captures parameters for one or more methods each time they are called. Parameters are logged inside the target application using its [`ILogger`](https://learn.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger).
+
+>**Note**: Unlike other artifacts, parameters do **not** support being sent to an egress provider.
+
+>**Note**: This feature is not enabled by default and requires configuration to be enabled. See [Enabling](#enabling) for more information.
+
+## Enabling
+
+This feature is currently marked as experimental and so needs to be explicitly enabled. The following configuration must be set for the feature to be used.
+
+```json
+"InProcessFeatures": {
+    "Enabled": true,
```

Comment:
This `Enabled` property is not strictly needed, it's sufficient to just set `InProcessFeatures:ParameterCapturing:Enabled` to `true`.